dissect/target/target.py

```diff
@@ -70,7 +71,7 @@ class Target:
         path: The path of a target.
     """
 
-    def __init__(self, path: str | Path | None = None):
+    def __init__(self, path: str | Path | None = None, minimal: bool = False):
```

Comment:
```suggestion
    def __init__(self, path: str | Path | None = None):
```
This is no longer necessary.